Documentation/guides/BuildProcess.md

```diff
@@ -631,6 +631,18 @@ when packaging Release applications.
 
     This property is `False` by default.
 
+-   **AndroidEnableProfiledAot** &ndash; A boolean property that
```

Comment:
These properties are now sorted alphabetically, as of 12fe161c1f16bf5608ef0f4f74d5347c88013b9f.

Please keep them alphabetically sorted. :-)